data/ShopifyDemoData.xml

```diff
@@ -1,43 +1,36 @@
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_CANCEL_ORDER_REMOTE" systemMessageTypeId="ShopifyCancelOrderMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/orders/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_CALCULATE_REFUND_REMOTE" systemMessageTypeId="ShopifyCalculateRefundMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/orders/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_REFUND_ORDER_REMOTE" systemMessageTypeId="ShopifyCreateRefundMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/orders/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_CREATE_FULFILLMENT_REMOTE" systemMessageTypeId="ShopifyCreateFulfillmentMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/fulfillments.json" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_GET_ADDRESSES_REMOTE" systemMessageTypeId="ShopifyGetCustomerAddressesMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/customers/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_GET_TRANSACTIONS_REMOTE" systemMessageTypeId="ShopifyGetOrderTransactionsMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/orders/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>
-    <moqui.service.message.SystemMessageRemote systemMessageRemoteId="DEMO_SHOPIFY_GET_FO_REMOTE" systemMessageTypeId="ShopifyGetFulfillmentOrderMessage" sendUrl="https://b6318f782c52bda768d23f5a404b51b6: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@moquidemo.myshopify.com/admin/api/2022-07/orders/" productStoreId="SHOPIFY_STORE" username="b6318f782c52bda768d23f5a404b51b6" password="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"/>

```

Comment:
I'm not sure what goes on at that url, but I wouldn't put that out there publicly

Reply:
Oh and the password... What are you doing there?

Reply:
nothing, the lines are deleted.